cargo/spec/fixtures/manifests/whitespace_names

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "dependabot" # the name of the package
+version = "0.1.0"    # the current version, obeying semver
+authors = ["support@dependabot.com"]
+
+[dependencies]
+    time = "0.1.12"
+  regex = "0.1.41"
```

Comment:
Maybe make one of these a tab?

Reply:
Hmm the time dependency should be a tab but it might have been converted to a space by git or your editor.
Is there a way to check what it is here? Maybe via the online editor?

Reply:
It looks like the online editor lets you choose between either tabs or spaces and converts them automagically for you 👎🏾

Reply:
Looks like it was spaces, but I converted it to a tab and pushed the change.